src/types/models/autoAlarm.ts

```diff
@@ -7,22 +7,22 @@ export interface DealerSavedSearchQuery {
   makeKey: string
   modelType: ModelType[]
   bodyType: string[]
-  firstRegistrationYearFrom: number | null
-  firstRegistrationYearTo: number | null
-  mileageFrom: number | null
-  mileageTo: number | null
-  priceFrom: number | null
-  priceTo: number | null
-  horsePowerFrom: number | null
-  horsePowerTo: number | null
+  firstRegistrationYearFrom?: number
```

Comment:
It was intentionally typed to be a number or null but not to be missing? I have in mind the application relies on this behavior as we run into issues with the form when the property was undefined. Also passing props form getServerSide props will have issues with undefined values as in `firstRegistrationYearFrom: undefined` will result in a serialization error... (U think it works fine if the property is not there...)

Reply:
hmm strange, I did not recall any form issues with listings where we use the optional types. I'll revert that change

Reply:
The one with the serialization is probably in with the new next version. Don't know about the listings, perhaps the approach to default the values is inclined in the form component... (Or I messed up something 🙃)

Reply:
I did a quick test and I was able to interact with saved searches properly with those new types. Unless QA discovers something that I missed, let's keep it the way it is.